buildpipeline/pipeline.json

```diff
@@ -33,11 +31,7 @@
           "Name": "Core-Setup-Linux",
           "Parameters": {
             "PB_DockerOS": "ubuntu.14.04",
-            "REPO_ID": "562fbfe0b2d7d0e0a43780c4",
-            "REPO_USER": "dotnet",
-            "REPO_SERVER": "azure-apt-cat.cloudapp.net",
-            "CLI_NUGET_FEED_URL": "https://dotnet.myget.org/F/cli-deps/api/v2/package",
-            "CLI_NUGET_SYMBOLS_FEED_URL": "https://dotnet.myget.org/F/cli-deps/symbols/api/v2/package"
```

Comment:
Yeah, the steps that use the values are already conditioned to only run on the correct platform, so passing them when they aren't used won't change any behavior.